web/src/app/selection/DestinationField.tsx

```diff
@@ -46,11 +65,7 @@ export default function DestinationField(
           props.onChange(newValues)
         }
 
```

Comment:
Same with this file- since it renders more than 1 field, let's make it plural: `DestinationFields`. This was a big source of confusion for me coming in new this quarter.